quidel_covidtest/delphi_quidel_covidtest/backfill.py

```diff
@@ -0,0 +1,121 @@
+# -*- coding: utf-8 -*-
+"""Store backfill data."""
+import os
+import glob
+from datetime import datetime
+
+import pandas as pd
+
+from delphi_utils import GeoMapper
+
+
+gmpr = GeoMapper()
+
+def store_backfill_file(df, _end_date, backfill_dir):
+    """
+    Store county level backfill data into backfill_dir.
+
+    Parameter:
+        df: pd.DataFrame
+            Pre-process file at ZipCode level
+        _end_date: datetime
+            The most recent date when the raw data is received
+        backfill_dir: str
+            specified path to store backfill files.
+    """
+    backfilldata = df.copy()
+    backfilldata = gmpr.replace_geocode(backfilldata, from_code="zip", new_code="fips",
+                          from_col="zip", new_col="fips", date_col="timestamp")
+    backfilldata.rename({"timestamp": "time_value",
+                         "totalTest_total": "den_total",
+                         "positiveTest_total": "num_total",
+                        "positiveTest_age_0_4": "num_age_0_4",
+                        "totalTest_age_0_4": "den_age_0_4",
+                        "positiveTest_age_5_17": "num_age_5_17",
+                        "totalTest_age_5_17": "den_age_5_17",
+                        "positiveTest_age_18_49": "num_age_18_49",
```

Comment:
I did. But it adds burn to the storage. But it might not be supper big deal.

Reply:
Having a consistent format is really nice and will save some work down the line if the backfill system gets more complicated. BUT either way works and this is just the first version (and I already have the backfill correction package to [handle quidel's nonstandard numerator/denominator names](https://github.com/cmu-delphi/covid-19/blob/7ba19ad8304199d269186b0679db15b230b69382/Backfill_Correction/delphiBackfillCorrection/R/main.R#L80-L86), so that's not a concern).